setup.py

```diff
@@ -42,8 +40,19 @@ def finalize_options(self):
         if not self.type in self.all_types:
             raise Exception('Test type must be [{0}]'.format(', '.join(self.all_types)))
 
+    @staticmethod
+    def _getTestEnv():
+
+        test_env = os.environ.copy()
+        ## Make sure that the PYTHONPATH is correct for the tests to pick up 'import Ganga'
+        test_sys_path = ''
+        for _dir in sys.path:
+            test_sys_path = test_sys_path + _dir + ':'
+        test_env['PYTHONPATH'] = test_sys_path
+
+        return test_env
+
```

Comment:
Do we actually need to do this? If the `cwd` of the `subprocess.check_call` is set to the `python` directory then we don't need to set it in the Python path since Python always looks in the CWD.